utility_analysis_new/tests/dp_engine_test.py

```diff
@@ -137,18 +153,21 @@ def test_utility_analysis_params(self):
                 budget_accountant = budget_accounting.NaiveBudgetAccountant(
                     total_epsilon=1, total_delta=1e-10)
                 options = utility_analysis_new.UtilityAnalysisOptions(
-                    epsilon=1, delta=0, aggregate_params=test_case["params"])
+                    epsilon=1,
+                    delta=0,
+                    aggregate_params=test_case["params"],
+                    pre_aggregated_data=test_case["pre_aggregated"])
                 engine = dp_engine.UtilityAnalysisEngine(
                     budget_accountant=budget_accountant,
                     backend=pipeline_dp.LocalBackend())
                 col = [0, 1, 2]
-
                 engine.analyze(col,
                                options,
                                test_case["data_extractor"],
                                public_partitions=test_case["public_partitions"])
 
-    def test_aggregate_public_partition_e2e(self):
+    @parameterized.parameters(False, True)
+    def test_aggregate_public_partition_e2e(self, pre_aggregated: bool):
```

Comment:
nit: are we testing analyze or aggregate? Is there a naming convention for the Python test names?
Should it be "test_analyze_applies_public_partitions"?